README.md

```diff
@@ -13,72 +14,115 @@
   <h1>PostCSS CLI</h1>
 </div>
 
-## Installation
+<h2 align="center">Install</h2>
```

Comment:
Well personally for github I think it introduces a clearer structure. npm is fine with it also and just left aligns them (ignores `align="center"`). Third opinion needed :)